cmd/telegraf/telegraf.go

```diff
@@ -281,6 +283,8 @@ func runAgent(ctx context.Context,
 		log.Printf("W! Deprecated outputs: %d and %d options", count[0], count[1])
 	}
 
+	_, _ = daemon.SdNotify(false, daemon.SdNotifyReady)
```

Comment:
Two questions:

1) For systems not running systemd will this have any issues?
2) Are there conflicts if the system is running telegraf as a system service and then as a binary as well?

Reply:
1. It should not according to docs. If the binary is not capable of Notify it's a null op.
2. Same here, should only be the service that have Type=notify that should send a proper Notify. If this were a problem systemd would not work at all.

Reply:
Thanks for confirming!